src/openforms/utils/tests/test_decorators.py

```diff
@@ -0,0 +1,62 @@
+from django import forms
+from django.contrib import admin
+from django.contrib.admin import AdminSite
+from django.test import RequestFactory, SimpleTestCase
+
+import requests
+
+from openforms.accounts.models import User
+from openforms.utils.decorators import suppress_requests_errors
+
+
+@suppress_requests_errors(User, fields=["first_name"])
+class TestAdmin(admin.ModelAdmin):
+    def formfield_for_dbfield(self, db_field, request, **kwargs):
+        if db_field.name in ["first_name", "last_name"]:
+            raise requests.RequestException
+
+        return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+
+class DbFieldsExceptionHandlerTests(SimpleTestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.request = self.factory.get("/test")
+
+    def test_none_crashing_field(self):
+        model_admin = TestAdmin(User, AdminSite(name="test"))
+
+        formfield = model_admin.formfield_for_dbfield(
+            User._meta.get_field("username"), self.request
+        )
+
+        self.assertIsInstance(formfield, forms.CharField)
+        self.assertNotIn(
+            "Could not load data - enable and check the request logs for more details",
+            formfield.widget.render("username", None),
+        )
```

Comment:
this assertion will pass when we add translations and the Dutch translation would be present.

Can't you rewrite this assertion as something like:

```py
self.assertHTMLEqual(
    rendered_widget,
    "<html here>",
)
```